langium/er-dsl/src/language/MetaModel/MultiRelationship.ts

```diff
@@ -55,17 +55,45 @@ export class MultiRelationship {
     public toString(): string {
         return this.simpleString();
     }
+    
+    //parse cardinality * -> N 
+    aggregateCardinality(cardinality: Cardinality): string {
+        if (cardinality == "*"){
+            return "N";
+        }
+        return cardinality.toString();
+    }
+
+    public toPlantUML(): string {
+        let result: string = '';
+
+        result += `relationship "${this.name}" as ${this.identifier} { }\n`
 
+        for (const connection of this.connections) {
+            const { entity, lower_cardinality, upper_cardinality, identifies } = connection;
+            let cardinality = lower_cardinality === upper_cardinality 
+            ? lower_cardinality
+            : `${lower_cardinality}..${upper_cardinality}`;
+            
+            result += `\n${entity.name} \-${this.aggregateCardinality(cardinality as Cardinality)}\- ${this.identifier}`;
```

Comment:
This breaks the type safety. The `cardinality` variable provided does not adhere to the `Cardinality` type. 
I suggest changing the argument type of the `aggregateCardinality` method.
That method could also be extracted to be a function, since it does not rely on access to the objects attributes.